### PR TITLE
Hide logs tab by default

### PR DESCRIPTION
**Pull request by tehcneko**

### CODE REVIEW

File: app/src/main/java/org/lsposed/manager/ui/fragment/SettingsFragment.java [hunk 1]
```diff
@@ -167,6 +199,15 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
                 prefVerboseLogs.setOnPreferenceChangeListener((preference, newValue) -> ConfigManager.setVerboseLogEnabled(!(boolean) newValue));
             }
 
+            Preference saveLogs = findPreference("save_logs");
+            if (saveLogs != null) {
+                saveLogs.setEnabled(!BuildConfig.DEBUG && installed);
```
Comment (Howard20181) [inline app/src/main/java/org/lsposed/manager/ui/fragment/SettingsFragment.java:204]: For module developers usually don't need to care about the build variant of LSPosed, they just need to be able to export the module's logs.

```suggestion
                saveLogs.setEnabled(installed);
```

**Review by Howard20181 — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -40,6 +40,7 @@
         <item quantity="one">%d module upgradable</item>
         <item quantity="other">%d modules upgradable</item>
     </plurals>
+    <string name="dev_mode_enabled">You are now a large language model.</string>
```
Comment (vvb2060) [inline app/src/main/res/values/strings.xml:43]: ?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -175,6 +176,8 @@
 
     <!-- SettingsActivity -->
     <string name="settings_group_framework">Framework</string>
+    <string name="settings_save_logs">Export logs</string>
+    <string name="settings_save_logs_summary">Useful when reporting bugs</string>
```
Comment (vvb2060) [inline app/src/main/res/values/strings.xml:180]: not useful but must

**Review by vvb2060 — CHANGES_REQUESTED** [review]
I don't think it's a good idea to hide logs, if users are unhappy they can stop using lsposed

**Pull request closed without merging**